Makefile

```diff
@@ -26,7 +26,7 @@ test: build
 	go test -v ./...
 
 $(OUTPUT): $(GO_SOURCES) vendor
-	go build riff.go
+	go build github.com/projectriff/riff-cli/cmd/riff
```

Comment:
@glyn Any particular reason for this approach?
I'd rather change it to be file based (and not package based) for consistency with all other Makefiles, wdyt?